<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KinectDialog</class>
 <widget class="QDialog" name="KinectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Base image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="image_rb">
       <property name="text">
        <string>Input image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="depth_rb">
       <property name="text">
        <string>Input depth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="detection_rb">
       <property name="text">
        <string>Detection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Features:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="blobs_cb">
       <property name="text">
        <string>Blobs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="centroids_cb">
       <property name="text">
        <string>Centroids</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="grid_cb">
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="boundaries_cb">
       <property name="text">
        <string>Boundaries</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Depth filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Min</string>
       </property>
       <property name="buddy">
        <cstring>min_hslider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="min_hslider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="min_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max</string>
       </property>
       <property name="buddy">
        <cstring>max_hslider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="max_hslider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="max_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OSC:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="server_le"/>
     </item>
     <item>
      <widget class="QCheckBox" name="send_osc_cb">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="image_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>image_label</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>boundaries_cb</zorder>
  <zorder>line_4</zorder>
 </widget>
 <tabstops>
  <tabstop>image_rb</tabstop>
  <tabstop>depth_rb</tabstop>
  <tabstop>min_hslider</tabstop>
  <tabstop>max_hslider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
